blogposts/2020/code-search-turned-code-checker.md

```diff
@@ -0,0 +1,296 @@
+contributions, e-mail me at <rijnard@sourcegraph.com>, I can help.
```

Comment:
@beyang I mentioned before this idea of "help people make OSS contributions using search results", and I've narrowed down the scope of search hits that I think we would be in a good position to do that. This section of the post is open-ended, i.e., I'm fine with people reaching out to me individually, but the other way that might work better is if we had a slack channel people could join (do we still have something like that)? How do you feel about this idea generally?

Reply:
I am in favor of this idea. It would be a great way to engage new users. I do not know if there is a way to enable a private Slack like ours have a public channel that anyone can join. Creating a separate community Slack channel poses the question of whether or not this would be visible enough to members of the engineering team.

What would be ideal is if we could enable anyone to join a community channel without approval, that channel is visible to members of the team, and we can restrict visibility of more sensitive channels to only Sourcegraph teammates.

Reply:
Didn't Sourcegraph have a kind of community slack in the past? Otherwise maybe we can just make a new one, like sourcegraph-community or something and link to our internal one, like support channels. @dadlerj any thoughts on this?

Reply:
I don't have a clear sense how to do this with slack, we can figure it out post publishing if it makes sense, for now people can just email me and/or I will advertise it on twitter.